code/modules/mob/living/carbon/xenomorph/castes/crusher/abilities_crusher.dm

```diff
@@ -119,12 +119,13 @@
 			to_chat(X, "<span class='xenowarning'>We try to fling [L] behind us, but there's no room!</span>")
 			return fail_activate()
 
-	//The target location deviates up to 1 tile in any direction
-	var/scatter_x = rand(-1,1)
+	//The target location deviates up to 1 tile in any direction //No.
+	/*var/scatter_x = rand(-1,1)
 	var/scatter_y = rand(-1,1)
 	var/turf/new_target = locate(T.x + round(scatter_x),T.y + round(scatter_y),T.z) //Locate an adjacent turf.
 	if(new_target)
 		T = new_target//Looks like we found a turf.
+	*/
```

Comment:
Fine I guess? Might want to talk this over with balance people

Reply:
removing this mostly due to inconsistency for both sides, sucks to be a xeno and your ability not do what you want and suck being marine and staying on a "safe spot" but RNG fucks you.